install_scripts.sh

```diff
@@ -8,6 +8,35 @@ display_help() {
   echo "  -h | --help            -  display this usage information"
 }
 
+file_changed_in_pr() {
+  local full_path="$1"
+  local base_branch="${2:-origin/2023.06-software.eessi.io}"  # Default to origin/2023.06-software.eessi.io
+
+  # Make sure file exists
+  [[ -f "$full_path" ]] || return 1
+
+  # Check if the file is in a Git repo (it should be) 
+  local repo_root
+  repo_root=$(git -C "$(dirname "$full_path")" rev-parse --show-toplevel 2>/dev/null)
+  if [[ -z "$repo_root" ]]; then
+    return 2  # Not in a git repository
+  fi
+
+  # Compute relative path to the repo root
+  local rel_path
+  rel_path=$(realpath --relative-to="$repo_root" "$full_path")
+
+  # Check if the file changed in the PR diff file that we have
+  (
+    cd "$repo_root" || return 2
+    if [ -f "$PR_DIFF" ]; then  # PR_DIFF should be set by the calling script
```

Comment:
Better make sure that `$PR_DIFF` is also defined, or the `-f` check may still succeed?

```suggestion
    # $PR_DIFF should be set by the calling script
    if [[ ! -z ${PR_DIFF} ]] && [[ -f "$PR_DIFF" ]]; then